src/index.js

```diff
@@ -136,10 +136,13 @@ class Griddle extends Component {
       }
     );
 
+    const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose
```

Comment:
@dahlbyk This breaks server side rendering because window is not available: ReferenceError: window is not defined

Reply:
Oh no! Should be easy to guard against with a `typeof window !== 'undefined'`, if you care to contribute?

Reply:
https://github.com/GriddleGriddle/Griddle/pull/765